src/lib.rs

```diff
@@ -1032,6 +1032,38 @@ impl<'a> WebViewBuilder<'a> {
   }
 }
 
+#[cfg(any(target_os = "macos", target_os = "ios",))]
+#[derive(Clone)]
+pub(crate) struct PlatformSpecificWebViewAttributes {
+  data_store_identifier: Option<[u8; 16]>,
+}
+
+#[cfg(any(target_os = "macos", target_os = "ios",))]
+impl Default for PlatformSpecificWebViewAttributes {
+  fn default() -> Self {
+    Self {
+      data_store_identifier: None,
+    }
+  }
+}
+
+#[cfg(any(target_os = "macos", target_os = "ios",))]
+pub trait WebViewBuilderExtDarwin {
+  /// Initialize the WebView with a custom data store identifier.
+  /// Can be used as a replacement for data_directory not being available in WKWebView.
+  ///
+  /// Only available on macOS >= 14 and iOS >= 17
```

Comment:
```suggestion
  /// - **macOS / iOS**: Available on macOS >= 14 and iOS >= 17
```

Just follow the other document's style